README.md

```diff
@@ -38,11 +38,11 @@ You will also see any lint errors in the console.
 
 ## Live Demo Link
 
-### [Live Demo]
+### [Live Demo](https://christabelle143-78cc21.netlify.app/)
 
 ## Loom Video Link
 
-### [Loom]
+### [Loom](https://www.loom.com/share/e826ba201e4146508c393a459515834c)
 
 
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **Nelson Sakwa**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.